packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -2011,6 +1993,22 @@ fn new_js_string<'s>(
     v8::String::new(scope, s).expect(&format!("Could not create JS String: {s}"))
 }
 
+/// Helper function to get the exception from a [`v8::TryCatch`]
+fn try_catch_exception<'s, 'p: 's, S>(try_catch: &mut v8::TryCatch<'s, S>) -> Option<String>
+where
+    v8::TryCatch<'s, S>: AsMut<v8::HandleScope<'p, ()>>,
+    v8::TryCatch<'s, S>: AsMut<v8::HandleScope<'p, v8::Context>>,
+    v8::TryCatch<'s, S>: AsMut<v8::Isolate>,
+{
+    let exception = try_catch.exception()?;
+    Some(
+        exception
+            .to_string(try_catch.as_mut())
```

Comment:
I feel like we should also read the exception message to provide more information.

Reply:
What do you mean by "read the exception message"? Add some `tracing`?

Reply:
In the documentation of `TryCatch` there is a method called [`message()`](https://docs.rs/rusty_v8/latest/rusty_v8/struct.TryCatch.html#method.message) which returns the exception message.

Reply:
Oh I see, it's weird that examples don't use it.
I added the message itself in https://github.com/hashintel/hash/pull/493/commits/14eeb4c6257453965bb1a9a1c703ab1d77617698 but we could display more than that, like the stack trace, the file name and line number, I don't know how much we want to print.